modules/tensor_mechanics/test/tests/plane_stress/weak_plane_stress_finite_tension_pull.i

```diff
@@ -0,0 +1,121 @@
+[GlobalParams]
+  order = FIRST
+  family = LAGRANGE
+  displacements = 'disp_x disp_y'
+  out_of_plane_strain = strain_zz
+[]
+
+[Problem]
+  extra_tag_vectors = 'ref'
+[]
+
+[Mesh]
+  [gmg]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 1
+    ny = 1
+  []
+[]
+
+[Variables]
+  [disp_x]
+  []
+  [disp_y]
+  []
```

Comment:
here, too

Reply:
I especially don't want to use add_variables for this case because this PR doesn't add support for it for the out of plane strain (because we're still doing some studies to find out what the best order/family to use is for that variable). I want to make it very clear here that we have 3 field variables.